app/form_models/jobseekers/search_form.rb

```diff
@@ -146,14 +166,16 @@ def set_organisation_type_options
     ]
   end
 
-  QuickApplyOptions = Struct.new(:value, :text, :hint, keyword_init: true)
+  # rubocop:disable Lint/StructNewOverride
```

Comment:
oops - I think this could all be avoided by using a Data type rather than a struct.

Reply:
Updated with a Data class :+1: